contracts/interfaces/IDCAKeep3rJob.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import 'keep3r-v2/solidity/interfaces/IKeep3r.sol';
+
+interface IDCAKeep3rJob {
+  /// @notice A struct that contains the swapper and nonce to use
+  struct SwapperAndNonce {
+    address swapper;
+    uint96 nonce;
+  }
```

Comment:
Would it make sense to add a `@dev` tag indicating max nonce? Or is it just too obvious?

Reply:
I would say it's unnecessary